provisioning/roles/webapp/templates/apache2.vhost.j2

```diff
@@ -1,14 +1,45 @@
 <VirtualHost *:80>
-    ServerName      Symfony
-    DocumentRoot    "{{ APPWEBROOT }}"
+    ServerName Symfony
+    ServerAlias Symfony
+
+    ErrorLog /var/log/apache2/project_error.log
```

Comment:
If you're gonna deviate from the main log, why not use APPNAME so it makes sense if you prov >1 app